feature/system/gnmi/cliorigin/mixed_oc_cli_origin_support_test/mixed_oc_cli_origin_support_test.go

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
I think you added this license manually because it uses the block styled comment. We should standardize on the line styled comments.

If you remove the license block, and subsequently use the `addlicense -l apache .` command, you should get the correct style.

Reply:
No actually I added it using addlicense only. Have changed it now.

Reply:
If the license was missing, addlicense should prefer line styled comment. But if you run addlicense on a file that already has a license, it will not do anything.

Like I said, you have to remove the license first, then rerun addlicense.

Reply:
Done. The previous comment came out before the git push which is why it was not visible